languages/go/exercises/concept/numbers-floating-point/.docs/introduction.md

```diff
@@ -0,0 +1,18 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+Different floating-point types can store different numbers of digits after the digit separator - this is referred to as its precision.
+
+Golang has 2 floating-point types:
+
+- `float32`: all IEEE-754 32-bit floating-point numbers. Written as `2.45`.
```

Comment:
I disagree as this is not an intro to numbers, but to floating point numbers. It needs to at least mention `float32`.

Reply:
Well, if `float32` is never used in Go, I'm concerned that including it in this exercise would actually make the exercise _less_ valuable. They say the secret of being a bore is tell _everything_; that's also the secret of being a bad teacher. At this early stage I think it's wise to keep the number of new concepts to a minimum, and only introduce those things which bring great value for the student in writing real programs.

Reply:
I don't see the harm in mentioning that it exists.

Note: I use float32 in my services as I have IOT devices that transmit float32. Also I have to build one of my applications for windows 32bit where float64 works but if its not necessary, why use it? So... I'm actually using float32 quite a bit. I'm not dismissing that the standard library expects float64 everywhere and that float32 is definitely not widespread. Still I think the information that a float32 exists doesn't hurt.

Reply:
Fair enough! Glad to hear there _is_ a use for it!

Reply:
> Note: I use float32 in my services as I have IOT and other hardware devices that transmit float32. Also I have to build one of my applications for windows 32bit where float64 works but if its not necessary, why use it? So... I'm actually using float32 quite a bit. I'm not dismissing that the standard library expects float64 everywhere and that float32 is definitely not widespread. Still I think the information that a float32 exists doesn't hurt.

It might be useful to add this information to the `after.md` document!